pkg/tarmak/provider/amazon/tf_remote_state.go

```diff
@@ -216,3 +218,25 @@ func (a *Amazon) validateRemoteStateDynamoDB() error {
 
 	return nil
 }
+
+func (a *Amazon) RemoteStateData(path string) (*terraform.State, error) {
```

Comment:
I would be prefer using here the logic coming from `terrafrom output`. That would make us more independet from cloud provider